Doppler.HtmlEditorApi/Domain/CampaignState.cs

```diff
@@ -12,4 +24,9 @@ public record CampaignState(bool OwnCampaignExists, bool ContentExists, int? Edi
         });
     public bool IsWritable => CampaignStatus.HasValue && WritableStatus.Contains(CampaignStatus.Value);
 }
-public record NoExistCampaignState() : CampaignState(false, false, null, null);
+public record NoExistCampaignState() : CampaignState(false, false, null, null, TestABCondition.NoTestAB);
+public record NoTestABCampaignState(
+    bool ContentExists,
+    int? EditorType,
+    CampaignStatus?
+    CampaignStatus) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition.NoTestAB);
```

Comment:
Related to https://github.com/FromDoppler/doppler-html-editor-api/pull/113/files#r872343322

```suggestion

public record NoTestABCampaignState(
    bool ContentExists,
    int? EditorType,
    CampaignStatus?
    CampaignStatus) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition.NoTestAB);
    
public record TestABCampaignState(
    bool ContentExists,
    int? EditorType,
    CampaignStatus?
    CampaignStatus,
    TestABCondition TestABCondition) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition);
```

Reply:
BTW, what do you think about calling it _Classic_ in place of _NonTestAB_?

Related to https://github.com/FromDoppler/doppler-html-editor-api/pull/113/files#r872343322

```suggestion

public record ClassicCampaignState(
    bool ContentExists,
    int? EditorType,
    CampaignStatus?
    CampaignStatus) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition.NoTestAB);
    
public record TestABCampaignState(
    bool ContentExists,
    int? EditorType,
    CampaignStatus?
    CampaignStatus,
    TestABCondition TestABCondition) : CampaignState(true, ContentExists, EditorType, CampaignStatus, TestABCondition);
```

Reply:
Looks good. Thanks for those suggestions.